Migrate testing to Github Actions and add support for installing on SUSE

Xorima opened the pull request:
### Description

Moves testing of linux to github actions
Creates a template for Windows testing

### Issues Resolved


### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

Xorima commented on README.md:
```diff
@@ -24,14 +28,15 @@ end
+- Windows 2016+
 
 ### Habitat
 
```
The line under this has `- Habitat version: 0.88.0` is this correct?

sam1el replied:
It is not. We are currently pinned to `1.5.0`. I plan to update that to current soon but, wanted to have a discussion about allowing it to always install current unless otherwise specified. 
@jonlives @tas50 I'd like your thoughts on that. I think we should be pushing latest and leave it on the customer to decide what version they want to pin to. Do we have some caveats to that?

sam1el commented:
looks like the dokken image for opensuse-15 is missing some dependencies required for install.

Xorima commented:
So from a local test I get this:

```
dokken login config-chef-13-auth-opensuse-leap-15
Directory: /
Fri Feb 28 21:05:06 UTC 2020
dokken:/ # curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  9420  100  9420    0     0  30387      0 --:--:-- --:--:-- --:--:-- 30289
--> hab-install: Installing Habitat 'hab' program
--> hab-install: Downloading via wget: https://packages.chef.io/files/stable/habitat/latest/hab-x86_64-linux.tar.gz
--> hab-install: Downloading via wget: https://packages.chef.io/files/stable/habitat/latest/hab-x86_64-linux.tar.gz.sha256sum
renamed '/var/tmp/tmp.CatK7xKucD/hab-latest.tar.gz' -> 'hab-x86_64-linux.tar.gz'
renamed '/var/tmp/tmp.CatK7xKucD/hab-latest.tar.gz.sha256sum' -> 'hab-x86_64-linux.tar.gz.sha256sum'
--> hab-install: GnuPG tooling found, downloading signatures
--> hab-install: Downloading via wget: https://packages.chef.io/files/stable/habitat/latest/hab-x86_64-linux.tar.gz.sha256sum.asc
--> hab-install: Downloading via wget: https://packages.chef.io/chef.asc
--> hab-install: GnuPG tooling found, verifying the shasum digest is properly signed
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: assuming signed data in 'hab-x86_64-linux.tar.gz.sha256sum'
gpg: Signature made Tue Feb 11 17:04:36 2020 UTC
gpg:                using DSA key 11685DB92F03640A2FFE7CA82940ABA983EF826A
gpg: /root/.gnupg/trustdb.gpg: trustdb created
gpg: Good signature from "CHEF Packages <packages@chef.io>" [unknown]
gpg:                 aka "Opscode Packages <packages@opscode.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 1168 5DB9 2F03 640A 2FFE  7CA8 2940 ABA9 83EF 826A
--> hab-install: Verifying the shasum digest matches the downloaded archive
hab-x86_64-linux.tar.gz: OK
--> hab-install: Extracting hab-x86_64-linux.tar.gz
xxx hab-install: Required command 'zcat' not found on PATH
```

What do I need to do to get it working @sam1el?
